<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Simon Persson</author>
 <class>BBMetaInfoDialog</class>
 <widget class="QDialog" name="BBMetaInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>284</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Song Tags</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="mPathLabel">
       <property name="text">
        <string>Path:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="mPathDisplay">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="mGuessButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Guess Values From File Name</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="mArtistLabel">
       <property name="text">
        <string>Artist:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>mArtistEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="KLineEdit" name="mArtistEdit">
       <property name="showClearButton" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="mAlbumLabel">
       <property name="text">
        <string>Album:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>mAlbumEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="KLineEdit" name="mAlbumEdit">
       <property name="showClearButton" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="mTitleLabel">
       <property name="text">
        <string>Title:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>mTitleEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="KLineEdit" name="mTitleEdit">
       <property name="showClearButton" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="mTrackLabel">
       <property name="text">
        <string>Track:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="KIntSpinBox" name="mTrackSpinner"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="mYearLabel">
       <property name="text">
        <string>Year:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>mYearSpinner</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="KIntSpinBox" name="mYearSpinner">
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="mGenreLabel">
       <property name="text">
        <string>Genre:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>mGenreEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="KLineEdit" name="mGenreEdit">
       <property name="showClearButton" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="mCommentLabel">
       <property name="text">
        <string>Comment:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QPlainTextEdit" name="mCommentEdit">
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="mButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mGuessButton</tabstop>
  <tabstop>mArtistEdit</tabstop>
  <tabstop>mAlbumEdit</tabstop>
  <tabstop>mTitleEdit</tabstop>
  <tabstop>mTrackSpinner</tabstop>
  <tabstop>mYearSpinner</tabstop>
  <tabstop>mGenreEdit</tabstop>
  <tabstop>mCommentEdit</tabstop>
  <tabstop>mButtonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>mButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>BBMetaInfoDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>BBMetaInfoDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
